test/views/CaptureWebsiteViewSpec.scala

```diff
@@ -87,6 +87,10 @@ class CaptureWebsiteViewSpec extends ViewBaseSpec {
           elementText(Selectors.removeWebsite) shouldBe "Remove website address"
         }
 
+        "have the correct remove website link" in {
+          element(Selectors.removeWebsite).attr("href") shouldBe "/vat-through-software/account/" +
```

Comment:
Could you change this to be the reverse controller route instead of a hardcoded String as it's a bit more 'safe' and demonstrates the relationship between controllers better